### PR TITLE
Tests: stub -> app

### PR DESCRIPTION
**Pull request by erikbern**
Easiest place to start since it doesn't affect docs or the public SDK

### CODE REVIEW

**Comment by erikbern** [comment]
@prbot approve

File: test/helpers.py [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
     file_or_module: str,
-    stub_variable: Optional[str] = None,
+    app_variable: Optional[str] = None,
     deployment_name="Deployment",
     cwd=None,
     env={},
     capture_output=True,
 ) -> Optional[str]:
-    # deploys a stub from another interpreter to prevent leaking state from client into a container process (apart from what goes through the servicer)
+    # deploys a app from another interpreter to prevent leaking state from client into a container process (apart from what goes through the servicer)
```
Comment (mwaskom) [inline test/helpers.py:18]: "a app" you say? ;)

Comment (mwaskom) [reply test/helpers.py:18]: (More seriously we should probably watch out for this when we update the user-facing docs)

**Review by mwaskom — COMMENTED** [review]
:+1: lot of changes but safe to rely on CI here

File: test/helpers.py (hunk 1 above)
Comment (erikbern) [reply test/helpers.py:18]: oops this was a mistake, but it's only in tests

Comment (mwaskom) [reply test/helpers.py:18]: yep not an issue in a test comment but good to think about (if you haven't been) for user-facing text

**Pull request merged**